### ISSUE TITLE
Prefixed Icon Links do not always colorize correctly when used within Brand Textboxes

### ISSUE THREAD

**Issue by wrenaria** [UCDavisLibrary/ucdlib-theme-wp#271]
There some funky colorization that happens if you put a prefixed icon link within a brand text box.

Examples on Stage at: https://stage.library.ucdavis.edu/brand-boxes/

Screenshots for quick ref/posterity:

![Screenshot 2023-12-19 at 2 48 14 PM](https://github.com/UCDavisLibrary/ucdlib-theme-wp/assets/17831969/6893e626-2c74-45cb-9949-ec1f9de684dd)

**Comment by spelkey-ucd** [comment]
@wrenaria Yeah, we're pushing on the limits of what the sitefarm theme can do. It isn't designed to allow brand colors within other brand colors. From a tech perspective, the `--category-brand` and `--category-brand-contrast-color` css variables get confused.

We ran into a similar issue with #259. For that, I was able to put in a little patch since, from a code perspective, the priority links are kind of a one-off. However, a full patch for this will likely be more involved.

But we might be able to figure something else out. What exactly are you trying to accomplish/how should it look? 

Do you just want an icon link to render in the brand textbox's normal colors? Like the datalab email link below:

![image](https://github.com/UCDavisLibrary/ucdlib-theme-wp/assets/34284106/f10bb01a-9f9a-443e-8dfc-06bcd733e09a)

Or do you actually want to use a different color for the icon while keeping the text the contrast color of the text box:
![image](https://github.com/UCDavisLibrary/ucdlib-theme-wp/assets/34284106/d3fe04c1-cd77-4357-b15f-30148eae5b16)

**Comment by wrenaria** [comment]
@spelkey-ucd It doesn't really matter which solution is used. All white works well and is the most forgiving/contrasty with the different colors, but it's not a currently achievable option from what is available. Ultimately, I just want it to be legible/consistently colored. A green text link on a green background is not.

But if it is a pain/theme breaking, I can figure out a different design solution/not use prefixed links on colored backgrounds.

**Comment by spelkey-ucd** [comment]
nah i can get it to be the contrast color (white for most), at least on the public side.
